dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
Yes, it's true we don't have these cleanly separated yet. Still I'm wondering what simple enough alternative options could we figure out for that specific case now.